mk/e2e.new.mk

```diff
@@ -13,6 +13,7 @@ test/e2e/kind/start/cluster/$1:
 		make kind/start
 	KIND_CLUSTER_NAME=$1 \
 		make kind/load/images
+	@kind load docker-image $(KUMA_UNIVERSAL_DOCKER_IMAGE) --name=$1
```

Comment:
why are we loading this image into kind?

Reply:
Oh this is our simulated external service, so we do not depend on "httpbin.org" for example.